mlrun/frameworks/_ml_common/pkl_model_server.py

```diff
@@ -56,3 +58,13 @@ def predict(self, request: dict) -> list:
         y_pred: np.ndarray = self.model.predict(x)
 
         return y_pred.tolist()
+
+    def explain(self, request: Dict[str, Any]) -> str:
+        """
+        Return a string explaining what model is being serve in this serving function and the function name.
```

Comment:
Returns a string listing the model that is being served in this serving function and the function name.